internal/cli/runner_install.go

```diff
@@ -268,6 +268,7 @@ func (c *RunnerInstallCommand) Run(args []string) int {
 		c.ui.Output("Error installing runner: %s", clierrors.Humanize(err),
 			terminal.WithErrorStyle(),
 		)
+		c.ui.Output(runnerInstallFailed, c.platform[0], id, terminal.WithWarningStyle())
```

Comment:
Pretty sure it should be this? Otherwise I think it's the same platform?

```suggestion
		c.ui.Output(runnerInstallFailed, platform, id, terminal.WithWarningStyle())
```

Or hrm, why is `c.platform` a slice string? Does runner install allow for installing to multiple platforms?

Reply:
@briancain it's a slice because this var is an `EnumVar` 😄 

```go
                f.EnumVar(&flag.EnumVar{
			Name:   "platform",
			Usage:  "Platform to install the Waypoint runner into. If unset, uses the platform of the local context.",
			Values: sortedPlatformNames,
			Target: &c.platform,
		})
```

Reply:
Strange! Yeah I see EnumVar's target is a slice, but it's only ever valid for 1 flag haha.